pkg/tools/spire/start.go

```diff
@@ -154,12 +163,16 @@ func Start(options ...Option) <-chan error {
 	}
 
 	// Healthcheck the Spire Agent
-	err = try.Do(func(attempts int) (bool, error) {
-		cmdStr := fmt.Sprintf("spire-agent healthcheck -socketPath %s", spireSocketPath)
-		return attempts < 10, exechelper.Run(cmdStr,
-			exechelper.WithStdout(log.Entry(opt.ctx).WithField("cmd", "spire-agent healthcheck").Writer()),
-			exechelper.WithStderr(log.Entry(opt.ctx).WithField("cmd", "spire-agent healthcheck").Writer()),
+	cmdStr = fmt.Sprintf("spire-agent healthcheck -socketPath %s", spireSocketPath)
+	_ = try.Do(func(attempts int) (bool, error) {
+		err = exechelper.Run(cmdStr,
+			exechelper.WithStdout(log.Entry(opt.ctx).WithField("cmd", "spire-agent healthcheck").WriterLevel(logrus.InfoLevel)),
+			exechelper.WithStderr(log.Entry(opt.ctx).WithField("cmd", "spire-agent healthcheck").WriterLevel(logrus.WarnLevel)),
 		)
+		if err != nil {
+			<-time.After(healthCheckTimeout)
+		}
+		return true, err
```

Comment:
Could we please just up the number of attempts here?  I really don't care how large we make them... but timers really really aren't the answer.   We see similar timing characteristics on the Github Actions CI (which is slow) and the Packet boxes (which should be fast... but have roughly the same spread of timing for the healthcheck we see in CI).

 This approach in particular is going to slow this down *everywhere*.  100ms is already the same size I see for *total* startup time in cmd-forwarder-vppagent, so this PR just literally *doubled* our startup time... for no real benefit I can see that can't be achieved by upping the count (a much simpler change).

Reply:
@edwarnicke why we can't just replace tries on `<-context.Done()`? In this case, we'll not care about the count of tries and our docker tests will able to set a custom expecting time for the startup of the spire. Is it make sense?

Reply:
@denis-tingajkin I like this idea very much :)

A simple loop with a

```go
for {
 exechelper.Run(
    fmt.Sprintf("spire-server healthcheck -registrationUDSPath %s/spire-registration.sock", spireRoot),
    exechelper.WithStdout(log.Entry(opt.ctx).WithField("cmd", "spire-server healthcheck").Writer()),
    exechelper.WithStderr(log.Entry(opt.ctx).WithField("cmd", "spire-server healthcheck").Writer()),
  )
  select {
  case <-opt.ctx.Done():
    errCh <- opt.ctx.Err()
    close(errCh)
    return errCh
  default:
}
```

would do it :)

Reply:
@denis-tingajkin Of course this means I now have to set a responsible timeout in my tests ;)